pkg/graph/resolvers/trb_request_relation.go

```diff
@@ -38,7 +38,11 @@ func SetTRBRequestRelationNewSystem(
 		}
 
 		// TODO: Store -> Delete CEDAR relationships
+
 		// TODO: Store -> Set contract number relationships
```

Comment:
Can remove this comment!

Reply:
(And others in this file, but not gonna spam comments for each)

Reply:
removed! Thanks!